app/services/cocina_support.rb

```diff
@@ -57,4 +57,26 @@ def self.update_version_and_lock(cocina_object:, version:, lock:)
   def self.collection_druid_for(cocina_object:)
     cocina_object.structural.isMemberOf.first
   end
+
+  # Returns the event date parsed from an EDTF date value.
+  # @param [Cocina::Models::DRO] cocina_object
+  # @param [String] type, e.g., 'publication'
+  # @return [Hash] with keys for year, month, and day
```

Comment:
Can also return `NilClass`, it looks like

Reply:
Fixed